patterns/sso-iam-identity-center/aws-auth/README.md

```diff
@@ -14,10 +14,11 @@ To do that use the link provided in the email invite - *if you added a valid ema
 
 With the active users, use one of the `terraform output` examples to configure your AWS credentials for SSO, as shown in the examples below. After you choose the *SSO registration scopes*, your browser windows will appear and request to login using your IAM Identity Center username and password.
 
-**Admin user example**
-```
+### Admin user example
+
+```bash
```

Comment:
just to be consistent, lets use `sh` like the rest of the patterns - change this throughout